bugbug/models/regressor.py

```diff
@@ -278,7 +278,7 @@ def evaluation(self) -> None:
         total_regressions = sum(1 for _, is_reg in results if is_reg)
         average_regression_rate = total_regressions / total_landings
 
-        logger.info("Average risk is %d", average_regression_rate)
```

Comment:
This is a valid fix but unrelated to the goal of this PR. We could open a new issue and fix it in a separate PR (with similar cases if there are any)

Reply:
Okay got it

Reply:
I filed a separate issue for this (i.e., #3837). This fix is not related to this PR. Please revert it.